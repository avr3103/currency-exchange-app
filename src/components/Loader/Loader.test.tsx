import React from 'react';
import { render } from '@testing-library/react';
import { Loader } from './index';

it('renders Loader', () => {
  const { asFragment } = render(<Loader />);
  expect(asFragment()).toMatchSnapshot();
});

