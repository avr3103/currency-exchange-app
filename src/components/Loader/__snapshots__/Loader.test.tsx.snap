// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Loader 1`] = `
<DocumentFragment>
  <div
    class="wrapper"
    data-testid="loader"
  >
    <div
      class="dual-ring"
    />
  </div>
</DocumentFragment>
`;
